### DISCUSSION TITLE
Why are there no tests?

### DISCUSSION THREAD

**Discussion by dugsmith** [waseefakhtar/dose-android#22] [Q&A] [answered]
Thank you for sharing your work while you develop this app as an attempt to showcase the latest best practices.

To that end: the first thing I look for in an app are the tests. Seeing none here, I'm wondering: do you have a plan to add them? Have you considered TDD?

**Comment by waseefakhtar** [comment, marked as answer]
Thanks for pointing that out!

I definitely have plans to add tests soon since that's part of showcasing the latest best practices. :)
